qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -133,6 +133,15 @@ def run(self,
                           PendingDeprecationWarning,
                           stacklevel=2)
             qobj = circuits
+            # Qobj values take precendence over run_options
+            # otherwise values from legacy assembly won't be preserved.
+            if not run_options:
+                run_options = qobj.config.__dict__
+            else:
+                run_options = copy.copy(run_options)
+                for key, value in qobj.config.__dict__.items():
+                    if key not in run_options and value is not None:
+                        run_options[key] = value
```

Comment:
The code doesn't seem to match the comment, here - things are only copied in from `qobj.config` if they don't exist in `run_options`, so they're not taking precedence.  Does the comment mean it's trying to take precedence over defaults?

Reply:
Ah good catch, I had to rework this a couple of times so it was backwards compatible with qobj options and tests passed, but forgot to update comment along with it.